gce-containers-startup/command/mock_runner.go

```diff
@@ -29,10 +29,11 @@ type MockCommand struct {
 }
 
 type MockRunner struct {
-	commands          map[string]*MockCommand
-	statFiles         map[string]os.FileInfo
-	expectedMkdirAlls map[string]bool
-	t                 *testing.T
+	FailOnUnexpectedCalls bool
```

Comment:
failOnUnexpectedCalls

Reply:
The reasons behind making this exported:
- I wanted a switch so that volume tests could be split between those that test errors and the returned map (from the function in volumes) and those that test the interaction with the OS. The first bunch should not have to set expectations on commands that will be run.
- The switch seemed like a legitimate candidate for a method of the MockRunner.
- I decided to make it exported instead of adding a method, since there is nothing else to do than just flip a flag.